# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/nsHttpConnection.h
# Commit: 521c9e019678
# Full Hash: 521c9e019678f584590f63473416d40e03f33342
# Author: Valentin Gosu <valentin.gosu@gmail.com>
# Date: 2023-03-15 04:10:55
# Regressor Bug: 1805372
# File Overlap Count: 1
# Description:
#   Bug 1805372 - Pass effectiveTRRMode to nsHttpChannel from socketTransport/DnsAndConnectSocket r=necko-reviewers,kershaw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D164539
# ==============================================================================

diff -r dfb39ca05529 -r 521c9e019678 netwerk/protocol/http/nsHttpConnection.h
--- a/netwerk/protocol/http/nsHttpConnection.h	Tue Mar 14 15:29:49 2023 +0200
+++ b/netwerk/protocol/http/nsHttpConnection.h	Tue Mar 14 12:59:40 2023 +0000
@@ -184,7 +184,8 @@
   nsresult GetPeerAddr(NetAddr* addr) override;
   bool ResolvedByTRR() override;
   bool GetEchConfigUsed() override;
-
+  nsIRequest::TRRMode EffectiveTRRMode() override;
+  TRRSkippedReason TRRSkipReason() override;
   bool IsForWebSocket() { return mForWebSocket; }
 
   // The following functions are related to setting up a tunnel.